charts/newrelic-k8s-metrics-adapter/values.yaml

```diff
@@ -62,17 +63,18 @@ resources:
 
 serviceAccount:
   # Specifies whether a ServiceAccount should be created for the job and the deployment.
-  create: true
+  create:
   # The name of the ServiceAccount to use.
   # If not set and create is true, a name is generated using the fullname template.
   name:
```

Comment:
Previously the defaults were explicit, but now they are not. Could we add a comment explaining them? Otherwise users might get confused.

Reply:
I've added the comments